### ISSUE TITLE
protobuf 23.3 no longer supports c++11

### ISSUE THREAD

**Issue by lilydjwg** [mobile-shell/mosh#1278]
```
checking whether protoc matches protobuf... no
In file included from /usr/include/absl/base/config.h:86,
                 from /usr/include/absl/base/attributes.h:37,
                 from /usr/include/absl/strings/string_view.h:39,
                 from /usr/include/google/protobuf/stubs/common.h:44,
                 from /usr/include/google/protobuf/io/coded_stream.h:130,
                 from conftest.pb.h:24,
                 from conftest.cpp:73:
/usr/include/absl/base/policy_checks.h:79:2: error: #error "C++ versions less than C++14 are not supported."
   79 | #error "C++ versions less than C++14 are not supported."
      |  ^~~~~
```

`export CPPFLAGS=-std=c++17` works.

**Comment by mahlzahn** [comment]
Workaround worked for me, too (protobuf version 23.4).

**Comment by achernya** [comment]
Fixed by eee1a8cf413051c2a9104e8158e699028ff56b26
